### PR TITLE
30 update homuntan02 about us

### LINKED ISSUE #30: Update homuntan02 About Us page
update about us and add picture

### PR DESCRIPTION
**Pull request by homuntan02**
## Which ticket does this address

- closes #30 

## What problems does this solve

- Missing image and role

## Solution/Changes

- Add image and role

## Checklist

- [x] All tests pass locally and on CI

### CODE REVIEW

File: docs/team/homuntan02.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+- **New Feature**: Added the ability to undo/redo previous commands.
+
+  - What it does: allows the user to undo all previous commands one at a time. Preceding undo commands can be reversed by using the redo command.
+  - Justification: This feature improves the product significantly because a user can make mistakes in commands and the app should provide a convenient way to rectify them.
+  - Highlights: This enhancement affects existing commands and commands to be added in future. It required an in-depth analysis of design alternatives. The implementation too was challenging as it required changes to existing commands.
+  - Credits: _{mention here if you reused any code/ideas from elsewhere or if a third-party library is heavily used in the feature so that a reader can make a more accurate judgement of how much effort went into the feature}_
+
+- **New Feature**: Added a history command that allows the user to navigate to previous commands using up/down keys.
+
+- **Code contributed**: [RepoSense link]()
```
Comment (yyj-02) [inline docs/team/homuntan02.md:21]: add your reposense link found here: https://nus-cs2103-ay2223s2.github.io/website/admin/teamList.html#cs2103t-w14

**Review by yyj-02 — CHANGES_REQUESTED** [review]
Can merge after this edit

File: README.md [hunk 2]
```diff
@@ -1,15 +1,14 @@
 [![CI Status](https://github.com/se-edu/addressbook-level3/workflows/Java%20CI/badge.svg)](https://github.com/se-edu/addressbook-level3/actions)
 [![codecov](https://codecov.io/gh/nus-cs2103-AY2223S2/tp/branch/master/graph/badge.svg?token=SNV76O467D)](https://codecov.io/gh/nus-cs2103-AY2223S2/tp)
-
 ![Ui](docs/images/Ui.png)
 
-* This is **a sample project for Software Engineering (SE) students**.<br>
+- This is **a sample project for Software Engineering (SE) students**.<br>
```
Comment (yyj-02) [inline README.md:5]: why you change it to `-`, should be `*`

**Review by yyj-02 — DISMISSED** [review]

**Review by clevon-w — APPROVED** [review]
You very handsome

**Pull request merged**